src/components/events/events.css

```
@@ -0,0 +1,48 @@
+.eventcard-grid {
+  display: grid;
+  grid-auto-flow: column;
+  border-radius: 10pt;
+  background-color: var(--events-bg-color);
+  width: 364px;
+  height: 98px;
+}
```

There is a problem when small mobile is selected. With grid-template-columns we don't need a fixed width anymore and the problem is solved.

```suggestion
.eventcard-grid {
  display: grid;
  grid-auto-flow: column;
  grid-template-columns: 2fr 1fr;
  border-radius: 10pt;
  background-color: var(--events-bg-color);
}
```

---

Great change for responsiveness! Didn't really get that. 🚀